fenix/app/src/test/java/org/mozilla/fenix/onboarding/view/OnboardingMapperTest.kt

```diff
@@ -18,29 +16,24 @@ class OnboardingMapperTest {
             imageRes = R.drawable.ic_onboarding_welcome,
             title = "default browser title",
             description = "default browser body with link text",
-            linkTextState = LinkTextState(
-                text = "link text",
-                url = SupportUtils.getMozillaPageUrl(SupportUtils.MozillaPage.PRIVATE_NOTICE),
-                onClick = stringLambda,
```

Comment:
The `stringLamda` property below is now used, let's delete that

Reply:
Good catch.  Thanks